plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel

```diff
@@ -1,11 +1,204 @@
 global proc AEmayaUsdProxyShapeTemplate( string $nodeName )
 {
-	editorTemplate -beginScrollLayout;
+    editorTemplate -beginScrollLayout;
 
-		// include/call base class/node attributes
-		AEsurfaceShapeTemplate $nodeName;
+        // Object Display (from AEdagNodeCommon.mel):
+        editorTemplate -beginLayout (uiRes("m_AEshapeTemplate.kObjectDisplay"));
```

Comment:
Is the uiRes call going to fail if AEdagNodeCommon.mel hasn't been sourced yet?  You were calling AEsurfaceShapeTemplate before (which calls a bunch of other base Templates).  Almost anything would end calling it, but if the AE is not being shown on startup and the first time it's shown we have a USD node selected...will it spit out an error?

Reply:
I tried your steps just to be sure. But even at startup the AE is empty and if I just create an empty stage that is the first object to be displayed in the AE.

uiRes is just a MEL proc that actually sources the correct xxx.res.mel file. It doesn't need AEdagNodeCommon.mel.